website/docs/cloud-docs/agents/hooks.mdx

```diff
@@ -42,14 +42,17 @@ Please note the following behavior when using hooks:
 	create a `pre-plan` and `pre-apply` hook, but you cannot create two
 	`pre-plan` hooks.
 - Each hook must have the execute permission set.
-- Each hook has a 60 second timeout. If a hook times out the Terraform run will
-	fail immediately.
-- Environment variables do not persist across hooks. For example, if a
+- Each hook has a default timeout of 60 seconds and a max timeout of 10 minutes. 
```

Comment:
I find parsing suggestions for lines hard to read, so putting this all in one for readability! I mainly added small suggestions for active voice and present tense:

```suggestion
Each hook has a default timeout of 60 seconds and a max timeout of 10 minutes. You can configure the timeout for each hook in seconds using an environment variable with the hook's name `TFC_AGENT_HOOK_[HOOK NAME]_TIMEOUT`. For example, you can set the timeout for the pre-plan hook to 2 minutes by setting `TFC_AGENT_HOOK_PREPLAN_TIMEOUT=120`. If a hook times out, the Terraform run fails immediately.
```

Reply:
Oh, and if you do like and commit this suggestion don't forget to remove all the lines below!

Reply:
This suggestion is wonderful!